### DISCUSSION TITLE
Go SDK: unable to get the actual error from container.WithExec()

### DISCUSSION THREAD

**Discussion by jritsema** [dagger/dagger#4003] [Help]
Hey folks.  I posted this issue before realizing that GitHub Discussions was enabled on this repo.

https://github.com/dagger/dagger/issues/3997

This gist is that I'm trying to run Terraform using Dagger and not getting underlying errors from the commands.

```
tf := client.Container().From("hashicorp/terraform:1.3.4")
tf = tf.WithMountedDirectory("/src", src).WithWorkdir("/src")

tf = tf.WithExec([]string{"init",
  fmt.Sprintf(`-backend-config='bucket=%s'`, bucket),
  fmt.Sprintf(`-backend-config='key=%s.tfstate'`, environmentName),
})
if out, err := tf.Stdout(ctx); err == nil {
  fmt.Println(out)
} else {
  return err
}
```

```
input:1: container.from.withMountedDirectory.withWorkdir.withExec.withExec.stdout process "/bin/terraform init -backend-config='bucket=123456789012' -backend-config='key=terraform.tfstate'" did not complete successfully: exit code: 1

Please visit https://dagger.io/help#go for troubleshooting guidance.
```

Is there some other way to get the actual error?

**Comment by nipuna-perera** [comment]
Hey there! There is a discussion on this in the help forum and some example workarounds - https://discord.com/channels/707636530424053791/827330142459396097

Related: #3192
